pages/understanding-json-schema/reference/type.md

```diff
@@ -175,3 +175,129 @@ types. For example, numeric types have a way of specifying a numeric
 range, that would not be applicable to other types. In this reference,
 these validation keywords are described along with each of their
 corresponding types in the following chapters.
+
+## Format[#format]
+
+The `format` keyword allows for basic semantic identification of certain
+kinds of values that are commonly used. or user could define its own format and use them alongside
+
+For example, because JSON doesn\'t have a \"DateTime\" type, dates need to be encoded as strings.
+`format` allows the schema author to indicate that the string value
+should be interpreted as a date. By default, `format` is just an
+annotation and does not effect validation.
+
+Optionally, validator [implementations](../../learn/glossary#implementation) can provide a configuration option
+to enable `format` to function as an assertion rather than just an
+annotation. That means that validation will fail if, for example, a
+value with a `date` format isn\'t in a form that can be parsed as a
+date. This can allow values to be constrained beyond what the other
```

Comment:
Suggestion:
This allows values to be constrained beyond what the other tools in JSON Schema can do, including Regular Expressions.